### PR TITLE
Make tracker.js more liberal in what it accepts

### PR DESCRIPTION
**Pull request by alicebartlett**
This PR is preparatory work for: https://www.pivotaltracker.com/story/show/93551518

This PR makes it possible to use Tracker.js with only one of the analytics codes (classic or universal). Tracker.js currently expects both a universal code and a classic code but we are switching off classic a few apps at a time so Tracker.js should be not expect both codes or need them in order to work.

This PR:

- normalises the interfaces of `google-analytics-classic-tracker.js` and `google-analytics-universal-tracker.js` so that `Tracker.js` can treat them both generically as trackers.
- Changes Tracker.js to accept one or both or none of the analytics ids.

@fofr it would be cool if you could take a look at this, but anyone else's reckons also welcome.

### CODE REVIEW

File: javascripts/govuk/analytics/tracker.js [hunk 1]
```diff
@@ -6,18 +6,25 @@
...
+    }
+    if (typeof config.classicId != 'undefined') {
+      this.trackers.push(new GOVUK.GoogleAnalyticsClassicTracker(config.classicId, config.cookieDomain));
+    }
   };
 
   Tracker.load = function() {
-    GOVUK.GoogleAnalyticsClassicTracker.load();
-    GOVUK.GoogleAnalyticsUniversalTracker.load();
+    for (var i=0; i < this.trackers.length; i++) {
```
Comment (fofr) [inline javascripts/govuk/analytics/tracker.js]: This method is called statically before a Tracker object is instantiated. So `this.trackers` won't exist at this stage.

Comment (alicebartlett) [inline javascripts/govuk/analytics/tracker.js]: Ah yeah, I got a bit carried away here didn't i... will amend. Thanks!

Comment (fofr) [inline javascripts/govuk/analytics/tracker.js]: Although this now means both the old and new GA libraries will always be loaded, even if they aren't needed.

**Comment by fofr** [comment]
A summary of a conversation with @alicebartlett
The dual loading of both GA libraries, even if only classic or universal is being used, is a concern. Avoiding the extra http hit to GA is preferable, while the behaviour of `ga.js` when loaded without any configuration is unknown – eg it could set cookies, though it probably doesn't.

In the context of this story https://www.pivotaltracker.com/n/projects/1261204/stories/93551518 (testing a deprecation of classic analytics) – it seems acceptable to continue to load both.

**Pull request merged**